docs/Contributing/Audit-logs.md

```diff
@@ -1208,6 +1208,23 @@ This activity contains the following fields:
 }
 ```
 
+## resent_configuration_profile_batch
```

Comment:
Just to be clear, this new activity will be:

* Global only (will _not_ show up in each of the hosts' past activities)
* Created only when the batch-resend endpoint is used (that is, we don't create it in the existing single-host profile resend endpoint)

Reply:
That's correct.

But now that you mentioned this, I'm now thinking we go ahead and add another activity for the single-host profile resend endpoint. How much effort would that be?

This way users aren't confused when they see the activity for the batch and not for the single resend.

Reply:
@mna that's correct.

But now that you mentioned this, I'm now thinking we go ahead and add another activity for the single-host profile resend endpoint. How much effort would that be?

This way users aren't confused when they see the activity for the batch and not for the single resend.

Reply:
If it's only for explicit resend (that is, whenever we hit the single-profile/single-host "resend" endpoint), that's not too much effort and we could add it in this story. If it should include whenever Fleet automatically resends something, that's more effort and should become it's own story.

Reply:
> If it's only for explicit resend (that is, whenever we hit the single-profile/single-host "resend" endpoint)

@mna this is what I was thinking.

But wait, it looks we already have this activity? https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Audit-logs.md#resent_configuration_profile